standardization/src/main/scala/za/co/absa/enceladus/standardization/StandardizationJob.scala

```diff
@@ -277,7 +276,7 @@ object StandardizationJob {
     stdRenameSourceColumns.setCheckpoint("Standardization - End", persistInDatabase = false)
 
     val recordCount = stdRenameSourceColumns.lastCheckpointRowCount match {
-      case None    => std.count
+      case None => std.count
```

Comment:
Why do these changes of removing spacing before the arrow?

Isn't it more standard to have `None` as a second/last option?

Reply:
1. The spacing must have been auto-adjusted, during refactoring I did some auto-format.
2. `None` as the last option is also the way I normally arrange things, but apparently it's not a convention. There's some `scalac` optimization for putting the happy path first, in addition to maybe doing `unapply` for `Some(x)` (I'm not sure about how true that is). But mostly I just kept the order the way it was before I touched it.